index.html

```diff
@@ -14,6 +14,57 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+
+  <nav>
+    <a href="#">Home</a>
+    <a href="#">About</a>
+    <a href="#">Portfolio</a>
+    <a href="#">Contact</a>
+  </nav> 
+  <!-- nav endse here -->
+  
+
+<div class="top-content">  
+<section class="intro">
+
+  <div class="logo">
+    <img src=images/mulan1.jpg alt="logo">
+</div>
+<div class="text">
+  <h1>Fa Mulan-Imperial agent Soldier Farm girl Warrior</h1>
+  <p>My name is Mulan and I starred in the 1998 American animated musical historical action adventure film produced by Walt Disney Feature Animation for Walt Disney Pictures. It is based on the Chinese legend of Hua Mulan, and was Disney's 36th animated feature and the ninth animated film produced and released during the Disney Renaissance.</p>
+</div>
+</section>
+<!-- .intro ends here -->
+</div>
+<!-- .top-content ends here -->
+
+<div class="middle-content">
+<section class="skills">
+  <h1> Here are my skills </h1>
+  <p>Trained warrior with the best that men of my time had to offer</p>
+  <p> Master of disguise. Pretty by day and pretty dangerous at all times!</p>
+  <p> Can carry a lot of milk jugs long distance.</p>
+
+</section>
+
+<section class="work-history">
+  <h1> Here is my work history</h1>
+  <p>Fought a war against the huns and won</p>
+  <p>Worked at a training camp amongst our best fighters</p>
+  <p> Can cleverly use a Huolongchushui cannon to cause an avalanche when desired</p>
+</section>
+</div>
+<!-- .middle-content ends here -->
+
+<div class="bottom-content">
```

Comment:
The bottom-content div is a little redundant here. You should be able to just have the section tag here